lib/restaurant.rb

```diff
@@ -0,0 +1,20 @@
+
+class Restaurant
+  def initialize ( messenger, menu)
```

Comment:
Style/SpaceAfterMethodName: Do not put a space between a method name and the opening parenthesis. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)